cmd/prometheus-config-reloader/main.go

```diff
@@ -105,6 +106,9 @@ func main() {
 	reloadURL := app.Flag("reload-url", "URL to trigger the configuration").
 		Default("http://127.0.0.1:9090/-/reload").URL()
 
+	reloadBasicUsername := app.Flag("reload-basic-username", "basic username passed as Authorization Header with Basic auth schema").Default("").String()
+	reloadBasicPassword := app.Flag("reload-basic-password", "basic password passed as Authorization Header with Basic auth schema").Default("").String()
```

Comment:
password should be read from a file.

Reply:
In my case I don't need it (because I pass flag with helm chart and terraform secret). Does there is a security problem behind this constraint ?
In case I would use a file does something like a plain file path containing a `user:password` or a more advanced json for exemple with 

```
{
   "user": "",
   "password": ""
}
```